specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -800,6 +800,11 @@ const queryComboBoxSpec = f.store(() =>
       syncers.maybe(syncers.toBoolean),
       syncers.default<boolean>(false)
     ),
+    showAddButton: pipe(
+      syncers.xmlAttribute('initialize addBtn', 'skip'),
+      syncers.maybe(syncers.toBoolean),
+      syncers.default<boolean>(true)
+    ),
```

Comment:
On line 813 in this file you will see `showNewButton`
Sp6 already supported hiding the add button. Please refactor this PR to use that instead of adding new property